package.json

```diff
@@ -1537,7 +1530,7 @@
                     },
                     "uniqueItems": true,
                     "markdownDescription": "%jupyter.configuration.jupyter.widgetScriptSources.markdownDescription%",
-                    "scope": "machine"
+                    "scope": "application"
```

Comment:
Seems like it would make sense as a workspace setting, since you could have widgets from different sources?

Reply:
i don't see why one would have widgets come from a cdn for one and not the other.